includes/admin/dashboard-widgets.php

```diff
@@ -153,42 +153,57 @@ function edd_load_dashboard_sales_widget( ) {
 
 		if ( $orders ) { ?>
 		<div class="table recent_purchases">
-			<table>
-				<thead>
-					<tr>
-						<td colspan="2">
-							<?php esc_html_e( 'Recent Purchases', 'easy-digital-downloads' ); ?>
-							&ndash;&nbsp;<a href="<?php echo esc_url( admin_url( 'edit.php?post_type=download&page=edd-payment-history' ) ); ?>"><?php esc_html_e( 'View All', 'easy-digital-downloads' ); ?></a>
-						</td>
-					</tr>
-				</thead>
-				<tbody>
-					<?php
-					foreach ( $orders as $order ) { ?>
-						<tr>
-							<td class="edd_order_label">
-								<a href="<?php echo add_query_arg( 'id', $order->ID, admin_url( 'edit.php?post_type=download&page=edd-payment-history&view=view-order-details' ) ); ?>">
-									<?php
-									$customer      = edd_get_customer( $order->customer_id );
-									$customer_name = $customer->name ? $customer->name : __( 'No Name', 'easy-digital-downloads' );
-									printf(
-										/* translators: 1. order number; 2. customer name */
-										esc_html__( '%1$s by %2$s', 'easy-digital-downloads' ),
-										esc_attr( $order->get_number() ),
-										esc_html( $customer_name )
-									); ?>
-								</a>
-							</td>
-							<td class="edd_order_price">
-								<a href="<?php echo add_query_arg( 'id', $order->ID, admin_url( 'edit.php?post_type=download&page=edd-payment-history&view=view-order-details' ) ); ?>">
-									<span class="edd_price_label"><?php echo edd_currency_filter( edd_format_amount( $order->total ), edd_get_payment_currency_code( $order->ID ) ); ?></span>
-								</a>
-							</td>
-						</tr>
+			<h3><?php esc_html_e( 'Recent Purchases', 'easy-digital-downloads' ); ?></h3>
+			<ul>
+			<?php
+			foreach ( $orders as $order ) {
+				$link = add_query_arg(
+					array(
+						'post_type' => 'download',
+						'page'      => 'edd-payment-history',
+						'view'      => 'view-order-details',
+						'id'        => $order->ID,
+					),
+					admin_url( 'edit.php' )
+				);
+				?>
+				<li class="edd_order_label">
+					<a href="<?php echo esc_url( $link ); ?>">
 						<?php
-					} // End foreach ?>
-				</tbody>
-			</table>
+						$customer      = edd_get_customer( $order->customer_id );
+						$customer_name = $customer->name ? $customer->name : __( 'No Name', 'easy-digital-downloads' );
+						$item_counts   = edd_get_order_item_counts( array( 'order_id' => $order->ID ) );
+						printf(
+							wp_kses_post(
```

Comment:
These should be switched to avoid code standard notices. `wp_kses_post()` needs to sanitize the entire output; right now it's not sanitizing any of the placeholder replacements.

Reply:
I am not getting notices with them in this order, but I can swap it to `echo wp_kses_post( sprintf(...` without error. With the `printf`, each item was being separately escaped.